docs/Release_Notes.md

```diff
@@ -4,9 +4,7 @@ GFS V16.3.22 RELEASE NOTES
 PRELUDE
 -------
 
-The upstream OBSPROC package is updated to v1.3. Along with this are the following companion updates:
-* workflow and UFS_UTILS package updates to use the new AFWA global snow file due to the hemispheric snow files being phased out
-* updated GSI code and convinfo file for saildrone observations
+The WW3 point output program (ww3_outp) is improved to process per-time-step point output to reduce the resources for the wave_post_pnt jobs.
```

Comment:
@KateFriedman-NOAA  did we need to add or replace this?

Reply:
The text should be added after the existing text in the prelude and the text on the prior lines should remain. It should essentially be text to say that this wave change is also being included.